components/stepper/stories/stepper.test.js

```diff
@@ -12,46 +12,43 @@ export const StepperGroup = Variants({
 			isQuiet: true,
 		},
 		{
-			testHeading: "Hide stepper",
-			hideStepper: true,
+			testHeading: "Invalid",
+			isInvalid: true,
 		},
-	],
-	stateData: [
 		{
-			testHeading: "Disabled",
-			isDisabled: true,
+			testHeading: "Quiet + invalid",
+			isQuiet: true,
+			isInvalid: true,
 		},
+	],
+	stateData: [
 		{
-			testHeading: "Focused",
-			isFocused: true,
+			testHeading: "Hide stepper",
+			hideStepper: true,
 		},
 		{
-			testHeading: "Keyboard-focused",
-			isKeyboardFocused: true,
+			testHeading: "Disabled",
+			isDisabled: true,
 		},
 		{
-			testHeading: "Invalid",
-			isInvalid: true,
+			testHeading: "Hovered",
+			isHovered: true,
 		},
 		{
-			testHeading: "Invalid + focused",
-			isInvalid: true,
+			testHeading: "Focused",
 			isFocused: true,
 		},
 		{
-			testHeading: "Invalid + keyboard-focused",
-			isInvalid: true,
+			testHeading: "Keyboard-focused",
 			isKeyboardFocused: true,
 		},
 		{
-			testHeading: "Invalid + disabled + focused",
-			isInvalid: true,
+			testHeading: "Disabled + hovered",
 			isDisabled: true,
-			isFocused: true,
```

Comment:
Did we want to keep `Disabled + focused`? It looks maybe the `Disabled + hover` replaced this on accident.

Reply:
I'll add back the stepper + focus; I’d removed it because focus doesn’t really have distinct styles so I thought we might get the same validation from hover.